src/app/(frontend)/embeds/courses/courses-date-filter.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+
+import { DateRangeFilter } from '@/components/filters/DateRangeFilter'
+import { QUICK_DATE_FILTERS_COURSES } from '@/constants/quickDateFilters'
+
+type Props = {
+  startDate: string
+  endDate: string
+}
+
+export const CoursesDateFilter = ({ startDate, endDate }: Props) => {
+  return (
```

Comment:
Can we just assign the filters here rather than in the `quickDateFilters` file?

```suggestion
  const QUICK_DATE_FILTERS = createQuickDateFilters('Past Courses')
  return (
```

Reply:
Yup!